src/index.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./components/app";
+import App from "./components/App";
```

Comment:
Curious why this change was needed? I had to change it back in order to launch the app locally.

Reply:
At General Assembly, our style guide has us use PascalCase for every component, even App.jsx, so I changed the file name out of habit. I've never had an issue with this before! It worked perfectly fine on my computer locally. Glad you were able to get it up and running, and sorry such an arbitrary changed caused an issue!